examples/hooks/components/Highlight.tsx

```diff
@@ -0,0 +1,100 @@
+import { Hit as AlgoliaHit } from '@algolia/client-search';
+import {
+  getHighlightedParts,
+  getPropertyByPath,
+} from 'instantsearch.js/es/lib/utils';
+import React from 'react';
+
+import { cx } from '../cx';
+
+type HighlightPartProps = {
+  children: React.ReactNode;
+  highlightedTagName: string;
+  nonHighlightedTagName: string;
```

Comment:
```suggestion
```suggestion
  highlightedTagName: React.ReactType;
  nonHighlightedTagName: React.ReactType;
```
```

Reply:
Thank you!! Was wondering if that type existed.

See 55f73bff7e078841577046ca321f5227cbe60514.